openlibrary/tests/solr/test_update_work.py

```diff
@@ -425,8 +427,21 @@ def test_ddcs(self, doc_ddcs, solr_ddcs, sort_ddc_index):
             assert 'ddc' not in d
             assert 'ddc_sort' not in d
 
-
-class Test_update_items():
+def mocked_urlopen(*args, **kwargs):
+    class MockResponse:
+        def __init__(self, json_data, status_code):
+            self.json_data = json_data
+            self.status_code = status_code
+        
+        def json(self):
+            return self.json_data
+        
+    if args[0] == "http://solr:8080/solr/select?wt=json&json.nl=arrarr&q=author_key:OL25A&sort=edition_count+desc&rows=1&fl=title,subtitle&facet=true&facet.mincount=1&facet.field=subject_facet&facet.field=time_facet&facet.field=person_facet&facet.field=place_facet":
+        return MockResponse({"facet_counts": {"facet_fields": {"place_facet": [], "person_facet": [], "subject_facet": [], "time_facet": []}},"response":{"numFound": 0}}, 200)
```

Comment:
```suggestion
    if args[0] == (
        "http://solr:8080/solr/select?wt=json&json.nl=arrarr&q=author_key:OL25A&"
        "sort=edition_count+desc&rows=1&fl=title,subtitle&facet=true&facet.mincount=1&"
        "facet.field=subject_facet&facet.field=time_facet&facet.field=person_facet&"
        "facet.field=place_facet"
    ):
        return MockResponse(
            {
                "facet_counts": {
                    "facet_fields": {
                        "place_facet": [],
                        "person_facet": [],
                        "subject_facet": [],
                        "time_facet": [],
                    }
                },
                "response": {"numFound": 0},
            },
            200,
        )
```

Reply:
python3 -m pip install black
black my_code_snippit.py  # will autoformat the code for readability.

Reply:
Ohh. Thanks!

Reply:
Done. @cclauss